app/qml/LoadingIndicator.qml

```diff
@@ -0,0 +1,81 @@
+import QtQuick 2.7
+import QtQuick.Controls 2.2
+import QtGraphicalEffects 1.0
+
+ProgressBar {
+    id: loadingIndicator
+    anchors.top: parent.top
+    value: 0
+    clip: true
+
+    property int easingType: Easing.InOutQuad
```

Comment:
would be good to add used colors as properties, so we could use this Item in theory elsewhere too